packages/server/tests/acceptance/index.spec.ts

```diff
@@ -139,7 +139,7 @@ describe('RPC Server Acceptance Tests', function () {
         // set env variables for docker images until local-node is updated
         process.env['NETWORK_NODE_IMAGE_TAG'] = '0.41.0-alpha.3';
         process.env['HAVEGED_IMAGE_TAG'] = '0.41.0-alpha.3';
-        process.env['MIRROR_IMAGE_TAG'] = '0.86.0-beta1';
+        process.env['MIRROR_IMAGE_TAG'] = 'main';
```

Comment:
what does the main does?

Reply:
We're pulling latest image that matches the mirror node main branch so we don't have to wait.
Once they tag a 0.88 rc we can bump it

Reply:
ohh thats great, and it makes sense now. Thanks for explaining! 👍